compose/.env-template

```diff
@@ -39,6 +39,9 @@ COLLAB_FQDN=collab.example.com
 INTEL_FQDN=intel.example.com
 INTEL_SECRET=super-secret-string
 
+# SSL cerfificate files should be placed in the `nginx/ssl` directory.
+# Ensure these files are present and correctly named.
```

Comment:
I do not think that this and the following comments are necessary. For simplicity, I would keep just the first comment.